tasks/main.yml

```diff
@@ -123,6 +123,20 @@
   when: elasticsearch_conf_dir is defined
   notify: Restart Elasticsearch
 
+# Create systemd override directory
+- name: Creating systemd override directory
+  file:
+    path: /etc/systemd/system/elasticsearch.service.d
+    state: directory
```

Comment:
Does this task also need a conditional? (to run only on systems that use systemd?)

Reply:
Thanks Hector, I added the conditional :)